frontend/public/components/_resource-dropdown.scss

```diff
@@ -23,6 +23,14 @@
   flex: 1 0 auto;
 }
 
+.co-search-group__accordian-label {
```

Comment:
Nit:  `accordian` should be `accordion`.

Reply:
nice